docs/en/samples/index.md

```diff
@@ -17,16 +17,19 @@ A Postman collection is also available at the bottom of the page.
 
 | Sample | Description |
 | :--- | :--------- |
```

Comment:
Thanks for making the copy. Could you rename to be `samples/jwt/` instead? Keep the documentation and samples within that directory.
Then keep this file focused on S2S like you are doing but avoid needing to add any JWT references.
In the top paragraph you can add a brief reference to the JWT samples.

Reply:
Maybe keep with the same file names for the samples and then it is obvious how to compare the two approaches

Reply:
the old samples do not refer to JWT workflow, rather they use different code to showcase the actions or GET calls, still relying on the config file, from which I moved away.
in my script files I try to follow this pattern:
-> provide input data
-> call the main function which will use
-->  in some cases a helper function to create the payload for the POST calls
-->  the call manager function to handle back-off and retry in all script files

If it's OK with you, I'll keep old scripts in the 'samples-old' folder for reference, but I'd not use them anymore (even delete them)

Reply:
I don't expect them to be used but best to keep them there for the time being. By marking them "jwt" it just groups that into that chapter of UMAPI. Then the current section can focus on the new way, but using S2S as the opportunity to do that.